src/product-item.css

```diff
@@ -0,0 +1,100 @@
+.product-item {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
+  font-family: "Roboto", Helvetica, sans-serif;
+  font-style: normal;
+  color: #060b35;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  padding: 32px 16px;
+
+  --main-color: #00acdc;
+}
+
+.img {
+  width: 160px;
+  height: 134px;
+  padding-left: 3px;
```

Comment:
why do you need this `  padding-left: 3px;`? maybe just center the image inside flex container?